Create Rule “do-you-add-https-to-text-fields/rule” (#4456)
SSWConsulting/SSW.Rules.Content: rules/do-you-add-https-to-text-fields/rule.md

---
type: rule
title: Do you add https:// to text fields and content?
uri: do-you-add-https-to-text-fields
authors:
  - title: Joseph Fernandez
    url: https://www.ssw.com.au/people/joe
created: 2022-11-30T00:13:40.254Z
guid: 4bf87ccf-91a1-470a-b1d6-bb2363761e06
---
Have you ever seen the text in an email or YouTube description meant to be a link to a page and wondered why it's missing the customary blue underline or can't be clicked? 

The answer is simple: the links are missing the prefix " https:// ", a prefix most modern content editors or browsers recognise as link identifiers and convert otherwise normal-looking text into easily clickable links.

![❌ Figure: Bad example the URL does not include https:// prefix. The URL is not short and readable.](url-https-image-1.png "Bad Example")

> Read the rule [Do you make URLs short and readable?](https://www.ssw.com.au/rules/create-friendly-short-urls) For a better understanding of how to create good URL's.

Reviewer: ```suggestion
Read the rule [Do you make URLs short and readable?](https://www.ssw.com.au/rules/create-friendly-short-urls) for a better understanding of how to create good URL's.
```

![✅ Figure: Good example. https:// prefix included making it a link. The URL has been made short and readable](url-https-image-2.png "SSW Video with a valid user link")

> Ultimately by simply adding the " https:// " text, you are encouraging more visitors to a link, as it has now become a lot easier for users to click the link and be directed to a page they are interested in.

Reviewer: ```suggestion
Adding " https:// " to the text makes the link easier to click which encourages more people to click on the link.
```
